src/clients/python/experimental_api_v2/library/grpcclient.py

```diff
@@ -1077,36 +1068,39 @@ def set_data_from_numpy(self, input_tensor):
         else:
             self._input.contents.raw_contents = input_tensor.tobytes()
 
-    def set_parameter(self, key, value):
-        """Adds the specified key-value pair in the requested input parameters
+    def set_data_from_shared_memory(self, region_name, byte_size, shape,
```

Comment:
set_shared_memory() ?
Remove shape and datatype as they should be in constructor.
Where is offset? It can default to 0 but it should be there and set the corresponding parameter.